src/components/Snackbar/SnackbarNotice.stories.tsx

```diff
@@ -0,0 +1,43 @@
+import { action } from "@storybook/addon-actions";
+import { Meta } from "@storybook/react";
+import { SnackbarNotice, SnackbarNoticeProps } from "src/components/Snackbar/SnackbarNotice";
+import { Css } from "src/Css";
+import { withBeamDecorator } from "src/utils/sb";
+
+export default {
+  component: SnackbarNotice,
+  title: "Components/Snackbar",
+  decorators: [withBeamDecorator],
+} as Meta;
+
+export function NoticeExamples() {
+  const propVariations: Partial<SnackbarNoticeProps>[] = [
```

Comment:
Nice, really like still having the "show all variations for visual regression" (along with the customizable / arg-driven story).